cafebabel/django_migrations.py

```diff
@@ -113,32 +117,51 @@ def handle_groups(groups):
                 'language': normalize_language(group_fields['language']),
                 'summary': group_fields['about'],
             }
+            if data['language'] == 'xx':
+                # It happens with unused tags(?)
+                continue
             try:
                 tag = Tag.objects.get_or_create(**data)
             except mongo_errors.NotUniqueError:
                 # We fallback on the old slug in that particular case.
                 data['slug'] = group_fields['slug']
                 tag = Tag.objects.get_or_create(**data)
             tags.append(tag)
-            if group_fields['feature'] == 'MAGAZINE':
-                category_slug = group_fields['slug']
-    return tags, category_slug
+    return tags
+
+
+def aggregate_gallery_body(gallery):
+    body = gallery['fields']['body']
+    for image in gallery['images']:
+        image_fields = image['fields']
+        image_src = normalize_image(image_fields['image'])
+        body += f'<p><img src="{image_src}" alt="{image_fields["title"]}"></p>'
+        body += image_fields['body']
```

Comment:
Wondering if we shouldn't got for
```html
<figure>
  <img src="{image_src}" alt="{image_fields["title"]}">
  <figcaption>{image_fields['body']}</figcaption>
</figure>
```

It could be easier to css tune old galleries later.

Reply:
Way better, thanks mate 👍